src/utils/formHelperFunctions.ts

```diff
@@ -76,6 +76,17 @@ function getDefaultFieldValue(field: any) {
   }
 }
 
+function updateStepFieldOptions(step: any, newFieldOptions: any) {
+  step.servar_fields.forEach((field: any) => {
+    const servar = field.servar;
+    if (servar.key in newFieldOptions) {
+      const options = newFieldOptions[servar.key];
+      servar.metadata.options = options;
+      servar.metadata.option_labels = options;
```

Comment:
Should we support setting option labels from the SDK as well?

Reply:
added